cmd/satellite/usage.go

```diff
@@ -54,7 +54,7 @@ func generateCSV(ctx context.Context, start time.Time, end time.Time, output io.
 	for _, row := range rows {
 		nid := row.NodeID
 
-		node, err := db.OverlayCache().Get(ctx, nid)
+		node, err := db.OverlayCache(0).Get(ctx, nid)
```

Comment:
This seems like a weird change when reading this code. Maybe instead add a method that changes the properties.

`db.OverlayCache().WithCacheDuration()`?

Reply:
Alternatively, maybe let's set some default cache and then have an explicit `Flush`/`Sync` call?

Reply:
I implemented your suggestion, with a method called `WithUpdateCache(updateCache UpdateCache) DB`. This solves two problems: allows me to not change the signature of the `OverlayCache()` method, and allows me to reuse the same cache between multiple `OverlayCache` calls, both of which were desirable.